webrtc/videoroom/config/runtime.exs

```diff
@@ -66,11 +79,23 @@ end
 
 # stun_servers: "addr:port"
 # turn_servers: "addr:port:username:password:proto"
-config :membrane_videoroom_demo,
-  stun_servers:
-    System.get_env("STUN_SERVERS", "64.233.163.127:19302") |> ConfigParser.parse_stun_servers(),
-  turn_servers: System.get_env("TURN_SERVERS", "") |> ConfigParser.parse_turn_servers(),
-  max_display_num: System.get_env("MAX_DISPLAY_NUM", "3") |> ConfigParser.parse_max_display_num()
+app_envs =
+  [
+    stun_servers:
+      System.get_env("STUN_SERVERS", "64.233.163.127:19302") |> ConfigParser.parse_stun_servers(),
+    turn_servers: System.get_env("TURN_SERVERS", "") |> ConfigParser.parse_turn_servers(),
+    max_display_num:
+      System.get_env("MAX_DISPLAY_NUM", "3") |> ConfigParser.parse_max_display_num()
+  ] ++
+    case System.get_env("MAX_PARTICIPANTS_NUM") do
+      nil ->
+        []
+
+      max_participants_num_raw ->
+        [max_participants_num: ConfigParser.parse_max_participants_num(max_participants_num_raw)]
+    end
```

Comment:
Instead of this case and merging two lists we can add a second clause for `parse_max_participants_num` 
```elixir
  def parse_max_participants_num(nil), do: nil
```
and then we can do
```elixir
config :membrane_videoroom_demo,
  stun_servers:
    System.get_env("STUN_SERVERS", "64.233.163.127:19302") |> ConfigParser.parse_stun_servers(),
  turn_servers: System.get_env("TURN_SERVERS", "") |> ConfigParser.parse_turn_servers(),
  max_display_num: System.get_env("MAX_DISPLAY_NUM", "3") |> ConfigParser.parse_max_display_num()
  max_participants_num: System.get_env("MAX_PARTICIPANT_NUM") |> ConfigParser.parse_max_participant_num()
  ```